pkg/daemon.py

```diff
@@ -82,6 +85,50 @@ async def get_options():
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e)) from e
 
+@app.get("/daemon/anomaly")
+async def daemon_anomaly( # pylint: disable = R0913
+    version: str = "4.15",
```

Comment:
lets default to 4.17 as that is the next in-dev release. Just to future proof things ;)